### DISCUSSION TITLE
Определить порядок элемента в списке

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#558] [Как это сделать?] [answered]
[totum.txt](https://github.com/totumonline/totum-ru-issues-and-discussions/files/15456382/totum.txt)
Добрый день!
Решал задачу: в таблице есть 5 полей и мне нужно получить кто из юзеров последний корректировал любое из этих полей.

Решил как во вложении.
	
Но что то мне подсказывает, что можно было сделать симпатичнее. Но не нашел как )) 
Буду признателен, если подскажете!

**Comment by erpic-ru** [comment, marked as answer]
Можно, например, сделать техническое поле, склеить в него кодом значения из 5 полей и смотреть только на его изменение. Либо кодом действия при изменении писать пользователя в отдельную колонку. Либо комбо из этих вариантов
